CodeReview/src/overlays/CodeReviewCommentOverlay.cpp

```diff
@@ -43,6 +43,8 @@ CodeReviewCommentOverlay::CodeReviewCommentOverlay(Visualization::Item* associat
 	Super{{associatedItem}, style}
 {
 	setAcceptedMouseButtons(Qt::AllButtons);
+	setFlag(QGraphicsItem::ItemIgnoresTransformations);
```

Comment:
If this works as you expected and you no longer need specialized behavior in `determineChildren` please remove that method.